simtel_pipeline.py

```diff
@@ -0,0 +1,292 @@
+import numpy as np
+import matplotlib.pyplot as plt
+# import digicampipe.io.hessio_digicam as hsm     #
+import astropy.units as u
+from digicampipe.io.event_stream import event_stream
+from digicampipe.utils import geometry
+from cts_core.camera import Camera
+from digicampipe.utils import utils
+from digicampipe.io.save_hillas import save_hillas_parameters_in_text, \
+    save_hillas_parameters
+
+from digicampipe.calib.camera import dl0, dl2, filter, r1, dl1
+from digicampipe.utils import utils, calib
+import simtel_baseline      #
+import events_image         #
+import mc_shower            #
+
+from optparse import OptionParser
+import os
+
+
+# import inspect
+# print(inspect.getfile(dl2))
+
+if __name__ == '__main__':
+
+    parser = OptionParser()
+    parser.add_option("-p", "--path", dest="directory",
+                      help="directory to data files",
+                      default='../../../sst-1m_simulace/data_test/ryzen_testprod/0.0deg/Data/')
+    parser.add_option("-o", "--output", dest="output",
+                      help="output filename", default="hillas", type=str)
+    parser.add_option('-c', "--camera_config",
+                      dest='camera_config_file',
+                      help='camera config file to load Camera()',
+                      default='digicampipe/tests/resources/camera_config.cfg')
+    parser.add_option('-i', "--picture_threshold", dest='picture_threshold',
+                      help='Picture threshold', default=15, type=int)
+    parser.add_option('-b', "--boundary_threshold", dest='boundary_threshold',
+                      help='Boundary threshold', default=7, type=int)
+    parser.add_option('-z', "--zenit", dest='zenit_angle',
+                      help='Zenit distance, THETAP',
+                      default=0, type=int)
+    parser.add_option('-a', "--azimut", dest='azimut', help='Azimut, PHIP',
+                      default=0, type=int)
+    parser.add_option('-r', "--primary", dest='primary',
+                      help='Primary particle', default='gamma', type=str)
+    parser.add_option('-d', "--baseline0", dest='baseline_beginning',
+                      help='N bins from the beginning of the waveform for \
+                      baseline calculation', type=int, default=9)
+    parser.add_option('-e', "--baseline1", dest='baseline_end',
+                      help='N bins from the end of the waveform for baseline \
+                      calculation', type=int, default=15)
+
+    (options, args) = parser.parse_args()
+
+    digicam_config_file = 'camera_config_digicampipe.cfg'
```

Comment:
this is overwritten at line 72 so no need for it here?

Reply:
Oh, sure, corrected..